cfme/tests/test_db_restore.py

```diff
@@ -0,0 +1,132 @@
+        vm.create_on_provider(allow_skip="default")
+    else:
+        logger.info("recycling deployed vm {} on provider {}".format(vm_name, provider_crud.key))
+    vm.provider_crud.refresh_provider_relationships()
+    vm.wait_to_appear()
+    return vm
+
+
+@pytest.fixture(scope="module")
+def get_appliances():
+    """Returns two database-owning appliances
+
+    """
+    ver_to_prov = str(version.current_version())
+    appl1 = provision_appliance(ver_to_prov, 'test_back')
+    appl2 = provision_appliance(ver_to_prov, 'test_rest')
+    appl1.configure(region=0, patch_ajax_wait=False)
+    appl1.ipapp.wait_for_web_ui()
+    appl2.configure(region=0, patch_ajax_wait=False)
+    appl2.ipapp.wait_for_web_ui()
+    return (appl1, appl2)
+
+
+def test_db_restore(request, provider_crud, provider_mgmt, soft_assert, roles):
+
+    appl1, appl2 = get_appliances()
+
+    def finalize():
+        appl1.destroy()
+        appl2.destroy()
+    request.addfinalizer(finalize)
+
+    appl1.ipapp.browser_steal = True
+    with appl1.ipapp:
+        # Manage infra,cloud providers and set some roles before taking a DB backup
+        config.set_server_roles(automate=True)
+        config.set_server_roles(ems_metrics_collector=True)
+        config.set_server_roles(ems_metrics_processor=True)
+        roles = config.set_server_roles()
+        provider_crud.create()
+        wait_for_a_provider()
+        setup_a_provider('cloud', 'ec2', validate=True)
+        cloud_provider.wait_for_a_provider()
+
+        providers_appl1 = appl1.ipapp.managed_providers
+        appl1.ipapp.backup_database()
+
+    source = appl1.ipapp.ssh_client()
+    dest = appl2.ipapp.ssh_client()
+
+    # Fetch v2_key and DB backup files from the source appliance
+    source_client = source(hostname=appl1.address)
+    dest_client = dest(hostname=appl2.address)
+    rand_filename = "/tmp/v2_key_{}".format(fauxfactory.gen_alphanumeric())
+    source_client.get_file("/var/www/miq/vmdb/certs/v2_key", rand_filename)
+    dest_client.put_file(rand_filename, "/var/www/miq/vmdb/certs/v2_key")
+    dump_filename = "/tmp/db_dump_{}".format(fauxfactory.gen_alphanumeric())
+    source_client.get_file("/tmp/evm_db.backup", dump_filename)
+    dest_client.put_file(dump_filename, "/tmp/evm_db.backup")
+
+    status, output = source.run_command('ls -l /tmp/evm_db.backup')
+    msg = 'Running ls on source,Status is {},output is {}'.format(status, output)
+    logger.info(msg)
+    status, output = source.run_command('md5sum /tmp/evm_db.backup')
+    msg = 'Running md5sum on source,Status is {},output is {}'.format(status, output)
+    logger.info(msg)
+
+    status, output = dest.run_command('ls -l /tmp/evm_db.backup')
+    msg = 'Running ls on dest Status is {},output is {}'.format(status, output)
+    logger.info(msg)
+    status, output = source.run_command('md5sum /tmp/evm_db.backup')
+    msg = 'Running md5sum on dest,Status is {},output is {}'.format(status, output)
+    logger.info(msg)
```

Comment:
From line 96 to here, I don't see status or output being used anywhere except the log lines. As far as I can tell, this section isn't doing anything and can be removed.